### DISCUSSION TITLE
[Supply] login credentials / review information for Android

### DISCUSSION THREAD

**Discussion by ItsMeSandu** [fastlane/fastlane#19225] [Ideas]
Hey there, 

the last "Google Play Developer Program Policy Update"-Mail i received told me that I have to provide login credentials for apps with login walls and in the play console I can add these under Policy > App content > App access. 

I studied the supply documentation a bit and could not found any information about the app access or review information. 
Did I overlook something or is this "feature" not given yet?

If not implemented: is this feature scheduled for a fastlane version?

looking forward to hearing from you :)

**Comment by ItsMeSandu** [comment]
Hey there, 
did my question reach you? :) 
I can't imagine that I'm the only one with this need.

**Comment by ItsMeSandu** [comment]
Still looking for some information or hints :)

**Reply by tylerablake** [reply]
Hey Sandu, do you have any update on this? My team is in need of this badly!

**Reply by ItsMeSandu** [reply]
sadly not, sorry.

**Comment by Alvarez215** [comment]
Much obliged to you for posting that it very well may be only what to offer motivation to somebody who needs it! Keep up the extraordinary work.

[CredibleBH](https://www.crediblebh.me/)
